content/source/docs/cloud/api/state-versions.html.md

```diff
@@ -593,98 +593,7 @@ curl \
-
-Parameter | Description
-----------|---------
-`:state_version_id` | The ID of the desired state version.
-
-Status  | Response                                     | Reason
---------|----------------------------------------------|----------
-[200][] | [JSON API document][]                        | Successfully returned a list of outputs for the given state version
-[404][] | [JSON API error object][]                    | State version not found, or user unauthorized to perform action
-
-### Query Parameters
-
-This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters). Remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
-
-Parameter                   | Description
-----------------------------|------------
-`page[number]`              | **Optional.** If omitted, the endpoint will return the first page.
-`page[size]`                | **Optional.** If omitted, the endpoint will return 20 state version outputs per page.
-
-### Sample Request
-
-```shell
-curl \
-  --header "Authorization: Bearer $TOKEN" \
-  --header "Content-Type: application/vnd.api+json" \
-  https://app.terraform.io/api/v2/state-versions/sv-SDboVZC8TCxXEneJ/outputs
-```
-
-### Sample Response
-
-```json
-{
-  "data": [
-    {
-      "id": "wsout-xFAmCR3VkBGepcee",
-      "type": "state-version-outputs",
-      "attributes": {
-        "name": "fruits",
-        "sensitive": false,
-        "type": "array",
-        "value": [
-          "apple",
-          "strawberry",
-          "blueberry",
-          "rasberry"
-        ]
-      },
-      "links": {
-        "self": "/api/v2/state-version-outputs/wsout-xFAmCR3VkBGepcee"
-      }
-    },
-    {
-      "id": "wsout-vspuB754AUNkfxwo",
-      "type": "state-version-outputs",
-      "attributes": {
-        "name": "vegetables",
-        "sensitive": false,
-        "type": "array",
-        "value": [
-          "carrots",
-          "potato",
-          "tomato",
-          "onions"
-        ]
-      },
-      "links": {
-        "self": "/api/v2/state-version-outputs/wsout-vspuB754AUNkfxwo"
-      }
-    }
-  ],
-  "links": {
-    "self": "https://app.terraform.io/api/v2/state-versions/sv-SVB5wMrDL1XUgJ4G/outputs?page%5Bnumber%5D=1&page%5Bsize%5D=20",
-    "first": "https://app.terraform.io/api/v2/state-versions/sv-SVB5wMrDL1XUgJ4G/outputs?page%5Bnumber%5D=1&page%5Bsize%5D=20",
-    "prev": null,
-    "next": null,
-    "last": "https://app.terraform.io/api/v2/state-versions/sv-SVB5wMrDL1XUgJ4G/outputs?page%5Bnumber%5D=1&page%5Bsize%5D=20"
-  },
-  "meta": {
-    "pagination": {
-      "current-page": 1,
-      "page-size": 20,
-      "prev-page": null,
-      "next-page": null,
-      "total-pages": 1,
-      "total-count": 2
-    }
-  }
-}
-```
+For more information please refer to the state version outputs documentation [here.](/docs/cloud/api/state-version-outputs.html#list-state-version-outputs)
```

Comment:
For various reasons, it's best to avoid having the text of a link be something like "here". (Screen readers present link text in isolation when you're jumping between links, so you want the text to actually say where the link goes.)

```suggestion
The output values from a state version are also available via the API. For details, see the [state version outputs documentation.](/docs/cloud/api/state-version-outputs.html#list-state-version-outputs)
```